content/general/channel_lifecycle.textile

```diff
@@ -0,0 +1,124 @@
+
+h2(#metadata-rest). Channel metadata REST API
+
+<%= partial 'shared/_channel_metadata' %>
+
+h2(#enumeration-rest). Channel enumeration REST API
+
+<%= partial 'shared/_channel_enumeration' %>
+
+h2(#reactor). Reactor rules
+
+You can associate "reactor rules":https://www.ably.io/reactor with metachannels in the same way as for regular channels; configure these using the "Message":/realtime/messages source and use a @channelFilter@ that matches the qualified metachannel name, or simply the name @[meta]channel.lifecycle@. The @Channel Lifecycle@ webhook source is no longer supported with the channel lifecycle metachannel.
+
+h1. Channel Lifecycle API Reference
+
+inline-toc.
+  Types:
+    - ChannelStatus#channel-status
+
+h2(#types). Types
+
+The primary payload of lifecycle events for channels is the "ChannelDetails":#channel-details type which contains the @channelId@ and other static information about the channel, plus a @status@ containing a "ChannelStatus":#channel-status instance which contains information about the current state of the channel.
+
+h3(#channel-details). ChannelDetails
+
+- channelId := the required name of the channel including any qualifier, if any<br>__Type: @string@__
+- region := in events relating to the activity of a channel in a specific region, this optionally identifies the region<br>__Type: @string@__
+- isGlobalMaster := in events relating to the activity of a channel in a specific region, this optionally identifies whether or not that region is resonsible for global coordination of the channel<br>__Type: @string@__
+- status := an optional "ChannelStatus":#channel-status instance<br>__Type: "ChannelStatus":/#channel-status__
+
+h3(#channel-status). ChannelStatus
+
+- isActive := a required boolean value indicating whether the channel that is the subject of the event is active. For events indicating regional activity of a channel this indicates activity in that region, not global activity<br>__Type: @boolean@__
+- occupancy := an optional "Occupancy":#occupancy instance indicating the occupancy of the channel. For events indicating regional activity of a channel this indicates activity in that region, not global activity.<br>__Type: "Occupancy":#occupancy__
+
+h3(#occupancy). Occupancy
+
+- metrics := an optional dictionary of membership categories for a realtime channel and their counts. Membership categories include:
```

Comment:
This is more of a question than anything else, but is it necessary to have three different types when the customer is really only ever seeing one type i.e. a JSON object with ChannelDetails, with a status, and an occupancy hash.  Could we not just show this as a single `ChannelDetails` object so that conceptually it's easier for a developer to udnerstand?  I am not sure this is the case, but it's a question worth consdiering./